StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1611ElementParametersMustBeDocumented.cs

```diff
@@ -84,6 +86,12 @@ protected override void HandleCompleteDocumentation(SyntaxNodeAnalysisContext co
             }
 
             var node = context.Node;
+            if (node.IsKind(SyntaxKindEx.RecordDeclaration))
```

Comment:
record structs have a separate SyntaxKind, so I suspect this will not behave as expected. I suggest you add a test for record structs as well, just to be sure.

Reply:
The test classes are inherited (and extended, when needed) from one language version to the next, so if you keep it in the c# 9 test project, it will still execute for c# 10 but in that situation it will run for record struct and record class as well, since the property will return all three in that project.

Reply:
Ok, I've made the CSharp9 test use the `CommonMemberData`.